src/icrar/leap-accelerate/algorithm/casa/PhaseRotate.cc

```diff
@@ -92,21 +88,27 @@ namespace casalib
                 metadata.GetBaselines(),
                 metadata.num_pols));
 
-            assert(48 == queue.front().data.dimension(0)); //metadata.channels
-            assert(8001 == queue.front().data.dimension(1)); //metadata.baselines
-            assert(4 == queue.front().data.dimension(2)); //metadata.polarizations
+#if _DEBUG
```

Comment:
Better to use the more standard `#ifndef NDEBUG`. And if the dumping to `std::cout` is removed there's not even a need for this, as `assert` is only enabled if `NDEBUG` is not defined.

Reply:
Note that `NDEBUG` should be used with `#ifndef` rather than `#ifdef`. One basically does a double negation and says "if this not a not debug build", which is confusing, but that's the standard way to go.